libamtrackForJs/.travis.yml

```diff
@@ -1,7 +1,7 @@
 sudo: required #is required to use docker service in travis
 services:
   - docker # required, but travis uses older version of docker :
-after_success:
+install:
```

Comment:
Take a look here:
https://docs.travis-ci.com/user/job-lifecycle/

especially on "Breaking the Build" and "The job lifecycle".

What about keeping `docker-compose build` in the "script" section to mark the job as failing and not errored in case of problem ?

I'd also move checking docker version to `before_script`.

Reply:
Done